app/src/main/java/mozilla/lockbox/view/RootActivity.kt

```diff
@@ -40,4 +40,15 @@ class RootActivity : AppCompatActivity() {
         super.onResume()
         presenter.onResume()
     }
+
+    override fun onStop() {
+        super.onStop()
+        presenter.onStop()
+    }
+
+    override fun onBackPressed() {
+        if (!presenter.onBackPressed()) {
+            super.onBackPressed()
```

Comment:
(if I'm understanding this correctly:) if the presenter doesn't have a specific implementation, use `super`'s?

Reply:
Correct.

Reply:
Yes. The activity's implementation will trigger the back button behaviour.